Resources/Prototypes/Entities/Objects/Specific/Robotics/borg_modules.yml

```
@@ -607,6 +608,7 @@
     - GasAnalyzer
     - HolofanProjectorBorg
     - GeigerCounter
+    - AtmosAlertsMonitorUnpowered
   - type: BorgModuleIcon
     icon: { sprite: Interface/Actions/actions_borg.rsi, state: rcd-module }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)